components/21-atoms/accolade/_accolade.scss

```diff
@@ -201,14 +203,14 @@ $accolade-height: 1rem;
   &.left-top,
   &.left-center,
   &.left-bottom {
-    left: -($accolade-height/2);
+    left: -($accolade-height*.5);
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.